mcs/class/System.Windows.Forms/System.Windows.Forms/XplatUICarbon.cs

```diff
@@ -2085,10 +2085,10 @@ internal override void SystrayBalloon(IntPtr hwnd, int timeout, string title, st
 		internal override bool Text(IntPtr handle, string text) {
 			Hwnd hwnd = Hwnd.ObjectFromHandle (handle);
 			if (WindowMapping [hwnd.Handle] != null) {
```

Comment:
Could you make a local constant for CFStringEncodingUTF8

Reply:
Fixed